pyteal/stack_frame.py

```diff
@@ -422,7 +489,6 @@ class PT_GENERATED:
     BRANCH_LABEL = "PyTeal generated branching label"
     TYPE_ENUM_TXN = "PyTeal generated transaction Type Enum"
     TYPE_ENUM_ONCOMPLETE = "PyTeal generated OnComplete Type Enum"
-    FLAGGED_BY_DEV = "Developer has flagged expression as compiler generated"
```

Comment:
qq: this `PT_GENERATED` reason deletion, and following deletion in `compiler_generated_reason`, are caused by removal of `_smap_friendly_approve` in router, right?

For it is the only place that mark `_compiler_gen` as `True`, which is the only cause of this reason?

Reply:
Yeah, I no longer was actually using the code, so deleted it.